codex-rs/core/tests/suite/exec_stream_events.rs

```diff
@@ -17,7 +17,7 @@ use codex_core::protocol::ExecCommandOutputDeltaEvent;
 use codex_core::protocol::ExecOutputStream;
 use codex_core::protocol::SandboxPolicy;
 
-fn collect_stdout_events(rx: Receiver<Event>) -> Vec<u8> {
+fn collect_stdout_events(rx: &Receiver<Event>) -> Vec<u8> {
```

Comment:
It's somewhat surprising that we don't transfer ownership in this case, but I guess this turns out to be consumed in `test_exec_stdout_stream_events_echo()`.

Maybe less surprising now that I realize this is a test.